packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -490,7 +490,7 @@ class SimcoreServiceLabels(DynamicSidecarServiceLabels):
     spec will be generated before starting the service.
     """
 
-    settings: Json[SimcoreServiceSettingsLabel] = Field(
+    settings: Json[SimcoreServiceSettingsLabel] | None = Field(
```

Comment:
I would avoid this change if possible. You made it nullable. Is that what youwanted to make or rather not-required?

Reply:
there are old services, such as the sleeper 1.0.0 that do not have any label set.

Reply:
I actually changed it now to not be None but to have a default_factory=dict and that better fits the needs.